### ISSUE TITLE
Email: Comments on WCAG 3.0 draft (8 - Role of User Testing)

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#487]
[Comment from Gregg Vanderheiden in Email](https://lists.w3.org/Archives/Public/public-silver/2021Mar/0002.html): 
from [Word docx](https://lists.w3.org/Archives/Public/public-silver/2021Mar/att-0002/Review_Comments_for_WCAG_3.0_-_Vanderheiden.docx) 

4)	Role and use of  User Testing

- User testing is really important for continual quality improvement (CQI).  
- User testing should be done on all websites and should involve a diversity of users.  
- User testing should be done on these guidelines to ensure that the guidelines themselves are as usable as possible.  
- User testing is important to carry out in the creation of guidelines. 
- But user testing is should not be used to test for conformance to a standard. 
 
User testing is not designed for and not a valid method for conformance testing  - as would be done with regulatory standards.  The reason is that user testing is not objective.  It would not have high interrater reliability.  That is, if you bring in 10 groups of users you are likely to come up with 10 different results (overlapping but not the same).  So, having an author do user-testing will not guarantee that anything will meet some objective level of accessibility.  In fact, I can assemble a group of individuals would like to use the web who likely would be able to fail any interface on the web including any interface designed by the working group.  I can also bring in a group that could use websites that fail WCAG. 

**Thus, user testing is very powerful and very good for what is it good at:**

- increasing the quality and usability of a site,  
- looking for missed issues
- identifying issues not covered by accessibility standards
  - in order to consider them for inclusion in a new standard – if they are objectively measurable 
  - in order to make a site more accessible than standards require (go beyond or fill in gaps)

But user testing does not have a place in testing for conformance to a standard.  It is not objective and does not have the necessary inter-rater reliability needed that an author could rely on it to answer their question “does this site pass”.  Nor to be used by an evaluator to prove that it does or doesn’t meet some stable, defined level of accessibility. 

**User testing does have a role in standards.**  

But it's in **the development** of standards **not in testing conformance** to standards.  Let me illustrate with an example that is a little easier to visualize.  
<hr>
Suppose you are creating standards for wheelchair access to bathrooms

You would not want some engineer to measure a wheelchair and  its turning radius, and then decide what the standard bathroom measurements should be for accessibility.

You would want to do a lot of user testing, where you had real people with disabilities, and real wheelchairs, navigating real bathrooms -  in order to understand what works and determine what the standard would be.

Once you have the standard, you would want it to be clear and objective so that the person building the bathroom could be held accountable to it.   They would knew exactly what the rules were, and could tell exactly if they had made met the criteria not.   It would not be up to inspectors to decide if it looked good enough – they would be able to (and be accountable for) making sure the clear rules were followed. 

If one used user testing to determine if the bathroom was accessible 

- Builders might choose one set of users – and find they could use a bathroom
- Only to find the inspector used a different set of users – and came to a different conclusion.  
- Different buildings would have different dimensions depending on which users were used to test and pass the bathrooms. 
- And if it ever ends up in court – (the only way to enforce regulations) then any testing based on individuals would fail as being non-objective and arbitrary based on which users are used.   And the court decides what is reasonable rather than the standard. 

<hr>

So user testing is not good for testing for conformance to  a standard.

However, as noted above, it IS good when developing a standard to help identify issues and determine what would be a reasonably effective threshold value for any measures (nothing is 100% good enough for all people). 

User testing is also very important for companies to use in testing the accessibility of their sites or products  – to figure out how to improve the accessibility of their sites, – to identify things they may have missed, and – to identify problems that may not have been covered (or might not have been able to be covered) by accessibility standards.

**Comment by jspellman** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by detlevhfischer** [comment]
I think it would be good to always add the commenter or commenting organisation in the issue name. Now you have to follow the link to the email archive to find out, which seems unnecessary.

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  The Accessibility Guidelines Working Group attempted to address these concerns in this latest draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
